src/js/schemas/job-schema/Schedule.js

```diff
@@ -35,7 +35,11 @@ const Schedule = {
     },
     timezone: {
       title: 'Time Zone',
-      description: 'Enter time zone in TZ format, e.g. America/New_York',
+      description: (
+        <span>
+          Enter time zone in <a href="http://www.timezoneconverter.com/cgi-bin/zonehelp" target="_blank">TZ format</a>, e.g. America/New_York.
```

Comment:
I went over with @sascala, and we decided to link to this as Wikipedia was not ideal. If you click on the country you get the TZ format.
I agree with you that this is not ideal either, but in any case helpful